resources/site-packages/quasar/library.py

```diff
@@ -0,0 +1,18 @@
+import xbmc
+import time
+import urllib2
+from quasar.config import QUASARD_HOST
+
+def library_thread():
+    limit = 8 * 60 * 60
+    count = limit - 60
+    while not xbmc.abortRequested:
+        # 8 hours passed - Update Library
+        if count >= limit:
+            count = 0
+            try:
+                urllib2.urlopen(QUASARD_HOST + "/library/update")
+            except:
+                pass
+        time.sleep(3)
+        count += 3
```

Comment:
If we're only updating the library every 8 hour (should that be a setting?), could we just sleep the thread longer? I doubt that'd have any impact on performance either way though.